main.go

```diff
@@ -12,6 +12,11 @@ import (
 var build = "0" // build number set at compile-time
 
 func main() {
+	// Load env-file if it exists first
+	if _, err := os.Stat(os.Getenv("PLUGIN_ENV_FILE")); err == nil {
+	  _ = godotenv.Load(os.Getenv("PLUGIN_ENV_FILE"))
```

Comment:
we could probably simplify:

```
if env := os.Getenv("PLUGIN_ENV_FILE"); env != "" {
   godotenv.Load(env)
}
```

I don't think there is any hard loading an env file that doesn't exist. It will just silently error which, in this case, is probably just fine